appveyor.yml

```diff
@@ -5,7 +5,7 @@ environment:
   matrix:
     - ARCH:                x64
       D_COMPILER:          dmd
-      D_VERSION:           2.077.1
+      D_VERSION:           2.078.3
```

Comment:
The current version used for bootstrapping is 2.079.1.

Reply:
Thanks! Updated.